resources/prosody-plugins/mod_muc_poltergeist.lua

```diff
@@ -410,8 +421,16 @@ function handle_create_poltergeist (event)
     else
         username = generate_uuid();
         store_username(room, user_id, username);
+        local context = {
+            user = {
+                id = user_id;
```

Comment:
is there nothing to set for the `avatar` and `name` fields?  i think i avoided assuming these were all present, but i didn't actually think there'd be a case that didn't have them.

Reply:
This information is already present in the same presence stanza, so no need for duplication.

Reply:
right but this is trying to mimic the `jitsi_meet_context_user` object, isn't it?  which has all of those fields...so things planning on reading that will expect them to be there.  like this they'll have to check for existence and, if they're not here, potentially read them from another location?

Reply:
Yep. Anyway, for identity, the main info is user_id and group_id, and the same data for the creator. Anything extra on top of that is not needed at the moment, but it is there, cause it is there and we copy the whole object :)